### PR TITLE
[T4A4][F11-B1] Le Minh Duc 

### PR DESCRIPTION
**Pull request by darrenlmd**

### CODE REVIEW

**Comment by darrenlmd** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```
Comment (Candiie) [inline src/seedu/addressbook/data/person/Printable.java:3]: Hi Hi, you are missing header comment here. Remember, all non trivial classes/methods should have javadoc format header comments.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -165,5 +166,18 @@ private static String getIndexedListForViewing(List<String> listItems) {
...
     }
+    
+    /**
+     * 
+     */
+    private String getPrintableString(Printable... printables) {
+    	StringBuilder builder = new StringBuilder();
+    	for (Printable printable: printables) {
+    		builder.append(printable.getPrintableString());
+    		builder.append(". ");
```
Comment (Candiie) [inline src/seedu/addressbook/ui/TextUi.java:177]: I think this result in an extra " " at the end. Take a look @ StringJoiner or String.join that was mentioned in tutorial :)

**Comment by Candiie** [comment]
Some comments added. Please close the PR after reading comments.

:+1: Awesome commits messages / practice! Keep it up! (Do note the comment on not including the prev activities commits ~ )

**Pull request closed without merging**